docs/Installing.md

```diff
@@ -34,6 +34,11 @@ Now that we've installed [*Redis*](https://redis.io), we can move on to installi
 
 `sudo apt install yarn`
 
+### Windows
+
+First download and install nodejs from [*this*](https://nodejs.org/dist/v16.2.0/node-v16.2.0-x64.msi) download link.
```

Comment:
Probably better to link to https://nodejs.org, since the version changes a lot and I don't want to send people directly to an install link.

Reply:
Okay